modules/lvms-creating-logical-volume-manager-cluster.adoc

```diff
@@ -72,7 +75,8 @@ When configuring multiple device classes, you must specify the device path for e
 <2> Set `fstype` to `ext4` or `xfs`. By default, it is set to `xfs` if the setting is not specified.
 <3> Mandatory: The `LVMCluster` resource must contain a single default storage class. Set `default: false` for secondary device storage classes.
 If you are upgrading the `LVMCluster` resource from a previous version, you must specify a single default device class.
-<4> Optional: To control what worker nodes the `LVMCluster` CR is applied to, specify a set of node selector labels.
+<4> Optional. To control or restrict the volume group to your preferred devices, you can manually specify the local paths of the devices in the `deviceSelector` section of the `LVMCluster` YAML. The `paths` section refers to devices the `LVMCluster` adds, which means those paths must exist. The `optionalPaths` section refers to devices the `LVMCluster` might add. You must specify at least one of `paths` or `optionalPaths` when specifying the `deviceSelector` section. If you specify `paths`, you do not have to specify `optionalPaths`. If you specify `optionalPaths`, you do not have to specify `paths` but at least one optional path must be present on the node. If you do not specify any paths, it will add all unused devices on the node.
```

Comment:
Can we update wordings for these statements, "If you specify `paths`, you do not have to specify `optionalPaths`" to "If you specify `paths`, it is not mandatory  to specify `optionalPaths`". Same for "If you specify `optionalPaths`, you do not have to specify `paths` but ...." to "If you specify `optionalPaths`, it is not mandatory to specify `paths` but ...."

Reply:
Done